sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-limit.sql

```diff
@@ -1,6 +1,9 @@
 -- A test suite for IN LIMIT in parent side, subquery, and both predicate subquery
 -- It includes correlated cases.
 
+-- Disable global limit optimization
```

Comment:
do we have a problem here?

Reply:
The golden result strictly requires returned values on limit query. This disables the optimization to get the required results.

Reply:
ah i see, thanks!